PyISY/Programs/__init__.py

```diff
@@ -192,7 +192,9 @@ def parse(self, xml):
                             plastrun = _empty_time
                         else:
                             plastrun = datetime.strptime(
-                                plastrun.toxml(), '%Y/%m/%d %I:%M:%S %p')
+                                plastrun.toxml(), '%Y/%m/%d %I:%M:%S %p'
+                                if plastrun.toxml().count(' ') == 2
+                                else '%Y/%m/%d %H:%M:%S')

```

Comment:
This in-line if took me a while to parse, especially because it uses the same `toxml` call passed as the first parameter to the method it's nested inside.

Let's split that out to increase readability